source/native/symbulation_sgp.cc

```diff
@@ -26,9 +26,9 @@ int symbulation_main(int argc, char * argv[])
 
   TaskSet task_set = LogicTasks;
   if (config.TASK_TYPE() == 0){
-     task_set = SquareTasks;
+     task_set = LogicTasks;
```

Comment:
This appears to be reversed from what we have in my repo

Reply:
Weird yeah the switch shouldn't make a difference so I'll just change it on my fork and accept this change